README.md

```diff
@@ -1,2 +1,63 @@
-# rfcs
-Repo for managing Requests For Comments (RFCs) for the ESLint project
+# ESLint RFCs
+
+Many changes, including bug fixes and documentation improvements, can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+However, some changes are "substantial", and we ask that these be put
+through a bit of a design process and produce a consensus among the ESLint Technical Steering Committee (TSC).
+
+The "RFC" (request for comments) process is intended to provide a
+consistent and controlled path for new features to enter the project.
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial"
+changes to any part of the ESLint project or its documentation. What constitutes a
+"substantial" change is evolving based on community norms, but may
+include the following.
+
+    - A new ESLint command line option.
+    - A new feature of ESLint core.
+    - A refactoring of existing ESLint core functionality.
+    - Any breaking change.
+
+In addition to these, the TSC may request an RFC for any other change that it deems "substantial" based on the size or scope of the request.
+
+If you submit a pull request to implement a new feature without going
+through the RFC process, it may be closed with a polite request to
+submit an RFC first.
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before diving into the
+level of API design detail required for an RFC. A good first step is to contact the [mailing list](https://groups.google.com/group/eslint) or [chatroom](https://gitter.im/eslint/eslint) to get thoughts from the community and the ESLint team before proceeding.
+
+## How to submit an RFC
+
+To submit a new RFC, follow these steps:
+
+1. [Fork](https://github.com/eslint/rfcs/fork) the RFC repo.
+1. Copy the appropriate template file from the `templates` directory into the `designs` directory.
+1. Rename the file to begin with the current year and include a meaningful description, such as `2018-typescript-support.md`.
```

Comment:
What's the goal of prefixing a proposal with the current year? Is it to make proposals be sorted in roughly chronological order?

[Rust](https://github.com/rust-lang/rfcs) identifies RFCs by giving them all a number, which happens to be equal to the number of the PR that created the RFC. I wonder if this would be a better way to to identify RFCs/sort them chronologically.

Reply:
I thought about that initially, but came to the conclusion that the date was a more useful piece of information when looking at a directory of proposals as it gives you some sense of how the project has developed and whether features are building on top of one another. 

I discovered this while trying to make sense of the proposals in the Ember directory. I just found it very hard to scan easily.